go/integrityblock/integrityblock_test.go

```diff
@@ -0,0 +1,85 @@
+package integrityblock
+
+import (
+	"bytes"
+	"encoding/hex"
+	"strings"
+	"testing"
+)
+
+func TestEmptyIntegrityBlock(t *testing.T) {
+	integrityBlock, err := GenerateEmptyIntegrityBlock()
+	if err != nil {
+		t.Errorf("GenerateEmptyIntegrityBlock. err: %v", err)
+	}
+
+	integrityBlockBytes, err := integrityBlock.CborBytes()
+	if err != nil {
+		t.Errorf("integrityBlock.CborBytes. err: %v", err)
+	}
+
+	expectedHex := "83 48 F0 9F 96 8B F0 9F 93 A6 44 31 62 00 00 80"
```

Comment:
You might want to use [testhelper.CborBinaryToReadableString](https://github.com/WICG/webpackage/blob/e4e2fab6b72c1940a7710ffaf203c50e3dd36010/go/internal/testhelper/cbor.go#L48) to verify CBOR bytes. You can see examples in `bundle/encoder_test.go` and `signedexchange/certurl/certchain_test.go`.